TradeBeacons/data/scripts/systems/tradingoverview.lua

```diff
@@ -20,24 +20,24 @@ function onInstalled(seed, rarity, permanent)
 
     if onServer() then
         tradingData = SimpleBuffer()
+        if historySize > 1 then
+            local sellable, buyable = gatherData()
+            tradingData:insert({sellable = sellable, buyable = buyable})
+            requestSectorsData()
+        end
     end
 end
 
 function getData()
     local sellable, buyable = gatherData()
 
-    if onServer() then
-        tradingData = SimpleBuffer()
```

Comment:
notice, we're not blowing away tradingData in getData any more. we'll continue to use that as the basis for data we're going to return